packages/tests/src/helpers/constants.ts

```diff
@@ -20,7 +20,12 @@ export const BUNDLER_VERSIONS: Record<BundlerFullName, string> = {
     webpack5: require('webpack5').version,
 };
 
-export const NO_CLEANUP = process.argv.includes('--no-cleanup');
+export const NO_CLEANUP = process.argv.includes('--cleanup=0');
```

Comment:
Apparently Jest doesn't allow custom flags without values.
`--no-cleanup` was automatically transformed into `--cleanup=0` by Jest's dependency `yargs` so it passed through the validation because it has a value in the end.

So I transformed it to `--cleanup=0` to have continuity with `--build=1` because simply `--build` wasn't possible because Jest falgged it 😢.